config/sets/doctrine-gedmo-annotations-to-attributes.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Php80\Rector\Class_\AnnotationToAttributeRector;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
```

Comment:
You need to use `RectorConfig` with `$rectorConfig->ruleWithConfiguration()` syntax

Reply:
Fixed!